### ISSUE TITLE
Logout Action Crash

### ISSUE THREAD

**Issue by jrom20** [dotnet-architecture/eShopOnContainers#544]
I just downloaded the branch xamarin-oidcclient (https://github.com/dotnet-architecture/eShopOnContainers/tree/xamarin-oidcclient) and I'm getting the following error when I log into the app and then I click the log out action see the images below to see what happen, I also noticed that I'm getting a prompt screen asking me if I would like to sign out, passing through the identity server log out method a null LogoutId value, any idea of why this is happening?

![image](https://user-images.githubusercontent.com/25846426/37412065-db75dc20-2769-11e8-9dbf-4de1f7dce2ea.png)

![image](https://user-images.githubusercontent.com/25846426/37412092-e43ebc3c-2769-11e8-902e-21b4c151ac09.png)

![image](https://user-images.githubusercontent.com/25846426/37412236-270c0e8e-276a-11e8-990e-e31d82db5afe.png)

![image](https://user-images.githubusercontent.com/25846426/37412117-eb5dcc74-2769-11e8-9cd7-1c60e35548e0.png)

**Comment by davidbritch** [comment]
Thanks for bringing this to my attention. At some point the backend was updated without the corresponding update being made to the Xamarin client.

I'd ignore the xamarin-oidcclient branch for now. I created that branch with the intention of immediately switching the Xamarin app to use the OIDCClient, and then had to go and work on other projects. Using the OIDCClient is one of the new features planned for v2 of the app/eBook.
